package.json

```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "main": "main.js",
   "scripts": {
-    "start:server": "nodemon main_node.js",
+    "start:server": "nodemon main.js",
```

Comment:
No, this is not right. The "main.js" here is the Electron one, the "main_node.js" is the one for the server. Please also restore the file.

Reply:
So the two files have diverged quite a bit. And it may be confusing later on to keep the two files in sync. 
For now, can we simply do something like: Have some comments on the main.js file, pointing to which part is related to electron and which is not? It will make it easier to copy paste the contents accordingly, until we try make them in sync automatically.

Reply:
Yeah, there is some issue there. I will try to have a shared common root there so that there are no such issues.